src/main/docker/Dockerfile

```diff
@@ -59,6 +56,12 @@ RUN chgrp 0 /usr/local/bin/docker-entrypoint.sh && \
 COPY scripts/ /opt/elasticsearch/scripts/
 RUN chmod +x /opt/elasticsearch/scripts/*
 
+# Configure logging
+COPY /maven/caf-*.jar /usr/share/elasticsearch/lib/
+COPY /maven/commons-*.jar /usr/share/elasticsearch/lib/
+COPY /maven/util-*.jar /usr/share/elasticsearch/lib/
+COPY /maven/log4j2.properties /usr/share/elasticsearch/config
```

Comment:
I don't understand why we're not just using the configuration from the jar.

Reply:
Elasticsearch is looking for the log4j.properties file to be in the ES 'config' folder